decisions/0003-github-labels-decluttering.md

```diff
@@ -0,0 +1,85 @@
+# Use of Labels in OSCAL Project(s)
+
+Date: 02/17/2023
+
+## Status
+
+Feedback Requested
+
+## Context
+
+We wish to improve communication within the team and among the community members. Labels we use in OSCAL repositories can be a part of the effort.  We have discussed a number of options around labels and we will use an ADR to capture and document our approach moving forward.
+
+- Related to Issue [#1496](https://github.com/usnistgov/OSCAL/issues/1496)
+
+## Decision
+
+Existing labels were reviewed, a subset of labels is kept and new ones are defined.
+
+The lists of labels that follow will use across OSCAL repositories.
+
+#### General labels:
+
+- breaking change
```

Comment:
This is a great start, thank you. As of the last few days, there was one I wanted to add that happened late after these conversations and I cannot recall immediately now. I will try to jump start my memory and update this with an additional recommendation once I recall. After that, I am ready to approve. (That, or I will need a follow-up PR like later today or the next day, seems silly on my part.)

Reply:
@aj-stein-nist I am ready to commit the label you want to add as soon as you remember it.